irs/src/components/Navigation/MainNav.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,} from 'reactstrap';
+
+class MainNav extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
this was probably just a leftover from your snippet shortcut, but you don't need props to be passed in since you're not using them anywhere in state